styles.css

```diff
@@ -0,0 +1,9 @@
+body {
+    font-family: 'Dosis', sans-serif;
+    color: navy;
+    font-size: 11px;
+    text-transform: lowercase;
+    text-align: justify;
```

Comment:
💡 As is, this rule isn't doing anything. You can comment it out and see, in the browser, that it has no effect. This may change, however, when you add `p` elements or other markup.